ANNOUNCE

```diff
@@ -1,4 +1,4 @@
-On behalf of the cclib development team, we are pleased to announce the release of cclib 1.7.1, which is now available for download from https://cclib.github.io. This is a major update to version 1.6.4 that drops support for Python 2, and includes several small improvements.
+On behalf of the cclib development team, we are pleased to announce the release of cclib 1.7.2, which is now available for download from https://cclib.github.io. This is a major update to version 1.6.4 that drops support for Python 2, and includes several small improvements.
```

Comment:
It is kind of odd to me that this still mentions 1.6.4. It doesn't need to be changed, but I didn't notice this for the 1.7.1 release.

Reply:
Indeed - removed the 1.6 mention and rephrased. This file is archaic, used to be used for sending the CCL announcement, and no one looks at it now -- it only functions for the release. We should consider getting rid of it.